osu.Game/Skinning/Editor/SkinEditorOverlay.cs

```diff
@@ -49,30 +49,12 @@ public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
 
                     Hide();
                     return true;
-
-                case GlobalAction.ToggleSkinEditor:
-                    Toggle();
-                    return true;
             }
 
             return false;
         }
 
-        public void Toggle()
-        {
-            if (skinEditor == null)
-                Show();
-            else
-                skinEditor.ToggleVisibility();
-        }
-
-        public override void Hide()
-        {
-            // base call intentionally omitted.
-            skinEditor?.Hide();
-        }
-
-        public override void Show()
+        protected override void PopIn()
```

Comment:
Since the logic has moved from `Show` to `PopIn`, scheduling inside of this method should no longer be necessary as they're invoked after the overlay is in a loaded state.